### PR TITLE
beta-feature: Resource sub-navigation

### PR DESCRIPTION
**Pull request by danharrin**
Documentation will be added when the design review is complete.

### CODE REVIEW

**Comment by archilex** [comment]
This appears to be a breaking change if you are already implementing getTabs() in a List*. 

https://flareapp.io/share/LPdg4zdm

**Comment by archilex** [comment]
When reviewing be sure to look at with `->sidebarCollapsibleOnDesktop()` as this also collapses the navigationItems

And pretty incredible feature! Well done!

File: packages/panels/resources/views/components/resources/tabs.blade.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@if (count($tabs = $this->getCachedTabs()))
+    @php
+        $activeTab = strval($activeTab);
```
Comment (drbyte) [inline packages/panels/resources/views/components/resources/tabs.blade.php:3]: ErrorException: `Undefined variable $activeTab`

Incomplete workaround from HasTabs trait: `$activeTab = $this->getDefaultActiveTab();`

**Comment by danharrin** [comment]
Yeah I fixed it

**Comment by danharrin** [comment]
> When reviewing be sure to look at with ->sidebarCollapsibleOnDesktop() as this also collapses the navigationItems

Thanks

**Pull request merged**